web-common/src/runtime-client/gen/index.schemas.ts

```diff
@@ -620,6 +623,11 @@ of in the runtime's metadata store. Currently only supported for the duckdb driv
   ingestionLimitBytes?: string;
 }
 
+export interface V1InlineMeasure {
+  name?: string;
+  expression?: string;
```

Comment:
Hey Brendan, thank you for taking a look! Some thoughts on your observations:

1. Totally agree on the optional fields not being ideal. Unfortunately, they're a consequence of the code generation pipeline that derives a Typescript client based on the API signatures. I can't find anything I can do about it just now (so think of these types more as autocomplete helpers than strict type checking), but will investigate separately if there are any ways we can improve the client codegen in general.

2. I added a whitelist on the backend that will currently only allow `COUNT(*)`, to prevent injection attacks.

3. To use the inline measures you're right that you have to provide a unique name for it client side. I would consider either generating a UUID or just using a name that's unlikely to collide, such as `__count`. Note that you also have to pass that name in the `measureNames` parameter.